src/components/Breadcrumbs/index.tsx

```diff
@@ -66,10 +74,10 @@ const BreadcrumbFragment: React.VFC<BreadcrumbFragmentProps> = ({
   );
 };
 
-const Container = styled.div`
+const Container = styled.nav`
   display: flex;
   align-items: center;
-  margin-bottom: ${size.l};
+  margin-bottom: ${size.s};
 `;
 
 const PaddedIcon = styled(Icon)`
```

Comment:
(placement of comment is random) Could we also prevent button text from overflowing? 🤔

<img width="404" alt="Screen Shot 2022-12-02 at 12 07 10 PM" src="https://user-images.githubusercontent.com/47064971/205346659-228daf1d-ddb1-4175-8495-e6cc9724658c.png">